src/styles/blocks/apply.scss

```
@@ -0,0 +1,144 @@
+.apply {
+  &__title-margin {
+    margin-bottom: 24px;
+  }
+
+  &__text {
+    margin-bottom: 64px;
+
+    font-family: Open Sans;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 150%;
+    text-align: center;
+    color: #6C788B;
+
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+
+    @include onDesktop {
+      display: block;
+    }
+  }
+
+  &__big-rectangle {
+    display: none;
+    position: absolute;
+    top: 124px;
+    width: 143.04px;
+    height: 56.54px;
+
+    opacity: 0.5;
+    border: 5px solid #2060F6;
+    box-sizing: border-box;
+    border-radius: 12px;
+    transform: rotate(-65.1deg);
```

```suggestion
    transform: rotate(-65deg);
```
Тут тоже будет достаточно целых значений